### PR TITLE
CIF-915 - Create JS unit tests for Navigation component

### PR DESCRIPTION
**Pull request by LSantha**
* added unit test for navigation component
 * improved clicking experience on down navigation arrow

## Related Issue

CIF-915

## How Has This Been Tested?

Manually, with unit tests.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by LSantha** [comment]
Ping.

File: ui.apps/test/components/commerce/navigation/navigationTest.js [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+        };
+        body = window.document.querySelector('body');
+        navigationRoot = document.createElement('div');
+        body.insertAdjacentElement('afterbegin', navigationRoot);
+        navigationRoot.insertAdjacentHTML(
+            'afterbegin',
+            `
+                <div class="header__primaryActions">
+                    <button class="navTrigger__root clickable__root" aria-label="Toggle navigation panel">
+                        <span class="icon__root">
```
Comment (dplaton) [inline ui.apps/test/components/commerce/navigation/navigationTest.js]: This is only a "test" HTML so I think you can trim it down a little bit. One example - you don't need the full SVG here. As a general rule, we only need the elements defined by a selector so we can simulate clicks.

File: ui.apps/test/components/commerce/navigation/navigationTest.js [hunk 2]
```diff
@@ -36,30 +36,23 @@ describe('Navigation', () => {
                 <div class="header__primaryActions">
                     <button class="navTrigger__root clickable__root" aria-label="Toggle navigation panel">
                         <span class="icon__root">
-                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
-                                <line x1="3" y1="12" x2="21" y2="12"></line>
-                                <line x1="3" y1="6" x2="21" y2="6"></line>
-                                <line x1="3" y1="18" x2="21" y2="18"></line>
+                            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (dplaton) [inline ui.apps/test/components/commerce/navigation/navigationTest.js]: >you don't need the full SVG here

What I meant to say is that you don't need the SVG *at all* here.

Comment (LSantha) [reply ui.apps/test/components/commerce/navigation/navigationTest.js]: Actually I need some svg at places, because I click on the svg in the test.

Comment (dplaton) [reply ui.apps/test/components/commerce/navigation/navigationTest.js]: But the actual event is not handled by the SVG, but by the `button` element, at least this is how it should be.

Comment (LSantha) [reply ui.apps/test/components/commerce/navigation/navigationTest.js]: Right, the event handler is registered on the button.

Comment (LSantha) [reply ui.apps/test/components/commerce/navigation/navigationTest.js]: @dplaton , do you need any more clarification?

**Pull request merged**